packages/devextreme/js/__internal/ui/map/m_map.ts

```diff
@@ -238,6 +249,11 @@ const Map = Widget.inherit({
       }
       case 'markerIconSrc':
         this._queueAsyncAction('updateMarkers', this._rendered.markers, this._rendered.markers);
+        break;
+      case 'googleMapConfig':
+        this._suppressAsyncAction = true;
```

Comment:
I didn't dive deep an reused the same option change handler as for provider runtime change. I allow to avoid sitiutions when there are some async actions like markers/routes updates are running